### PR TITLE
improvement: mermaid mix task for ER diagrams

### PR DESCRIPTION
**Pull request by joshprice**
- Use PDF instead of PNG (for smallest vector form)
- Improve docs
- Refactor flow chart mix task for consistency

# Contributor checklist

- [x] Documentation changes
- [x] Refactoring

### CODE REVIEW

File: lib/mix/tasks/generate_er_diagrams.ex [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    config =
+      if File.exists?("mermaidConfig.json") do
+        "--configFile #{Path.expand("mermaidConfig.json")}"
+      end
+
+    "sh"
+    |> System.cmd([
+      "-c",
+      """
+      cat <<EOF | mmdc --output #{file} #{config} --scale 10
```
Comment (zachdaniel) [inline lib/mix/tasks/generate_er_diagrams.ex:68]: do we still need the `--scale 10` option w/ it as PDF?

Comment (joshprice) [reply lib/mix/tasks/generate_er_diagrams.ex:68]: nope

File: lib/mix/tasks/generate_er_diagrams.ex [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  end
+
+  def sibling_file(file) do
+    __ENV__.file
+    |> Path.dirname()
+    |> Path.join(file)
+  end
+
+  def apis do
+    Mix.Project.config()[:app] |> Application.get_env(:ash_apis, []) |> dbg
```
Comment (zachdaniel) [inline lib/mix/tasks/generate_er_diagrams.ex:90]: `dbg` 😆

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**